order/order.bal

```diff
@@ -0,0 +1,385 @@
+import ballerina/sql;
+import ballerinax/mysql;
+import ballerina/http;
+import ballerina/time;
+
+configurable string USER = ?;   
+configurable string PASSWORD = ?;
+configurable string HOST = ?;
+configurable int PORT = ?;
+configurable string DATABASE = ?;
+configurable string CONSUMER_ENDPOINT = ?;
+configurable string RESTAURANT_ENDPOINT = ?;
+configurable string MENU_ITEM_ENDPOINT = ?;
+configurable string ACCOUNTING_ENDPOINT = ?;
+configurable string DELIVERY_ENDPOINT = ?;
```

Comment:
Shall we make variable names camel cases? 
since these are not constants, better we change them to camel case